### DISCUSSION TITLE
hexo init errno: -4048

### DISCUSSION THREAD

**Discussion by czx-cn** [hexojs/hexo#4849] [Question / Answer]
* os : win11
* node.js ,git checked all in C
* if hexo init in C ,D,H will sucess,
but in F  `:git clone failed. Copying data instead
FATAL {
  err: [Error: EPERM: operation not permitted, mkdir 'F:\'] {
    errno: -4048,
    code: 'EPERM',
    syscall: 'mkdir',
    path: 'F:\\'
  }
} Something's wrong. Maybe you can find the solution here: %s http://hexo.io/docs/troubleshooting.html`

**Comment by yoshinorin** [comment]
Please check your `F` drive's permission.
This problem is out of hexo.
